websites/A/atomicradio/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "904084297831571518",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			smallImageKey: "search",
```

Comment:
Use image links or [provided assets](https://github.com/PreMiD/Presences/blob/c598cf31f004d621445e57a67b41c158b291fd2f/%40types/premid/index.d.ts#L78-L96) over Discord asset names.
```suggestion
			smallImageKey: Assets.Search,
```